expression/builtin_op.go

```diff
@@ -377,7 +385,8 @@ func (b *builtinRightShiftSig) evalInt(row chunk.Row) (int64, bool, error) {
 
 type isTrueOrFalseFunctionClass struct {
 	baseFunctionClass
-	op opcode.Op
+	op       opcode.Op
+	castNull bool
```

Comment:
Please add some comments for this field.

Reply:
Is it better to keep it false builtin.go? Would it be a better option to naming it `keepNull` ?

Reply:
updated